postfix/edition15.html

```diff
@@ -1401,20 +1406,25 @@ <h5>Configuration</h5>
 exit;</code>
 
 	<p>
-		Next is to edit the MySQL's <b><i>my.cnf</i></b> file.
-		In Ubuntu/debian this is created by default.
-		In Mandrake I had to manually create a blank one in /etc.
-		But we need to configure it, so:
-		<code>sudo vi /etc/mysql/my.cnf</code>
-		In previous versions you needed to comment out this line
-		<code class="document"><span class="comment">#skip-networking</span></code>
-		However in today's file the default is to bind the
-		address to localhost, which is fine.
+		Next is to create a specific config MySQL config file.
+		Before we start adding configuring options, please double check that
+		the bind-adress value is <b>not</b> hashed out:
+		<code>
```

Comment:
@flurdy  Please check the formatting here, a little bit uncertain.

Reply:
@flurdy  As I suspected, the formatting was wrong.

![image](https://user-images.githubusercontent.com/4511254/101160079-b0f25880-362e-11eb-889c-d7a4da66637d.png)

Maybe you can fix it in another PR?

Thanks!